package Learn;

public class exception {

	public static void main(String[] args) {
	
		
		try {
			int y=10/0;
			
		} 
		
	
			catch (ArithmeticException e)	
		{
				
				
	           	e.printStackTrace();
	}
		catch (Exception e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
			


}
}
}